javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,21 @@
+var fullName = 'Katherine Bustamante';
+var isCool = true;
+var totalPets = 0;
+
+console.log('value of fullName:', fullName);
+console.log('value of isCool:', isCool);
+console.log('value of totalPets:', totalPets);
+console.log('type of fullName:', typeof fullName);
+console.log('type of isCool:', typeof isCool);
+console.log('type of totalPets:', typeof totalPets);
+
+var unknown = null;
+
+console.log('value of unknown:', unknown);
+console.log('type of unknown:', typeof unknown);
+
+// eslint-disable-next-line no-unused-vars
+var never =
```

Comment:
Don't disable my eslint rules please. This should be `var never;`, `var never =` is a syntax error, and will give you a red error in your console.